ov/firmware_drivers.go

```diff
@@ -145,6 +147,30 @@ func (c *OVClient) GetFirmwareBaselineById(id string) (FirmwareDrivers, error) {
 	return firmwareId, nil
 }
 
+func (c *OVClient) GetFirmwareBaselineByNameandVersion(name string) (FirmwareDrivers, error) {
+	fwNameVersion := strings.SplitAfter(name, ",")
+	if len(fwNameVersion) < 2 {
```

Comment:
This indicates user need to provide both name and version, why is so? I mean name should be main criteria but version could be optional right